js/tuid-helper.js

```diff
@@ -53,7 +53,7 @@ function addTuidRow(tuidtype, tuid, localdate){
     tr.appendChild(type);
     tr.appendChild(value);
     tr.appendChild(local);
-    tbody.appendChild(tr);
+    tbody.insertBefore(tr, tbody.firstChild);
 }
 
 function addTuid(tuidtype, tuid) {
```

Comment:
I was also thinking about changing how TUID gets inserted as well. Instead it adding it at the end using push, use unshift to insert it first. This will match the array order with display order. Also the removal process has to be updated as well. Instead of shift, it will be pop. Similar situation will apply to updating html table rows. Remove last instead of first.